packages/adminrouter/extra/src/lib/service.lua

```diff
@@ -107,6 +108,36 @@ local function resolve_via_marathon_apps_state(service_name, marathon_cache)
     return true, nil, nil
 end
 
+local function requrl_rewriting_required(service_name, marathon_cache)
```

Comment:
I'm not very sure of the name here. `requrl` doesn't seem to follow the pattern anywhere else.

Reply:
Addressed in https://github.com/dcos/dcos/pull/2733